newrelic/resource_newrelic_workflow.go

```diff
@@ -31,7 +31,7 @@ func resourceNewRelicWorkflow() *schema.Resource {
 				Required:    true,
 				Description: "(Required) The name of the workflow.",
 			},
-			"destination_configuration": {
```

Comment:
If you're making changes to the schema you will need to add schema migrations: https://www.terraform.io/plugin/sdkv2/resources/state-migration

The reason is that the customer who are already using this will have a statefile with the old field names. I'd suggest we also provide a migration guide with steps on what changes to make. 

There's an example of a schema migration here: https://github.com/newrelic/terraform-provider-newrelic/blob/main/newrelic/resource_newrelic_alert_policy_channel.go#L48

Reply:
Done